servicetalk-client-api-internal/src/test/java/io/servicetalk/client/api/internal/AbstractRequestConcurrencyControllerMultiTest.java

```diff
@@ -34,8 +34,8 @@
 import static org.hamcrest.Matchers.is;
 
 public abstract class AbstractRequestConcurrencyControllerMultiTest {
-    @Rule
-    public final Timeout timeout = new ServiceTalkTestTimeout();
+    @RegisterExtension
+    public final ExecutorExtension<Executor> exec = ExecutorExtension.withCachedExecutor();
```

Comment:
Why this is necessary? `exec` is not used in this test class

Reply:
You are right. It is not needed. I removed it.